tests/Doctrine/Performance/Hydration/MixedQueryFetchJoinArrayHydrationPerformanceBench.php

```diff
@@ -7,11 +7,10 @@
 use Doctrine\DBAL\Result;
 use Doctrine\ORM\Internal\Hydration\ArrayHydrator;
 use Doctrine\ORM\Query\ResultSetMapping;
+use Doctrine\Performance\ArrayResultFactory;
 use Doctrine\Performance\EntityManagerFactory;
-use Doctrine\Tests\Mocks\ArrayResultFactory;
 use Doctrine\Tests\Models\CMS\CmsPhonenumber;
 use Doctrine\Tests\Models\CMS\CmsUser;
-use PhpBench\Benchmark\Metadata\Annotations\BeforeMethods;
```

Comment:
The `BeforeMethods` annotation is used in this and other files you've changed. The import should be kept.

Reply:
Thanks. They got removed somehow automatically, probably by the PhpStorm's Optimize imports.